devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -132,6 +138,8 @@ def levelEventHandler(currentLevel) {
 	if (lastLevel == "undefined" || currentLevel == lastLevel) { //Ignore invalid reports
 		log.debug "Ignore invalid reports"
 	} else {
+		state.invalidSameLevelEvent = true
+		currentLevel = currentLevel < 0 ? 0 : currentLevel > 100 ? 100 : currentLevel
```

Comment:
Please, could You explain why did You add this line ? Do Your devices report such values that need to be constrained that way?

Reply:
Are You saying that it's useful when You ask Alexa or Google to set values 0 <  and > 100 or does it happen in other conditions ?